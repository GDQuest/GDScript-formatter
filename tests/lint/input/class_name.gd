#region Good
class_name GoodClassName
#endregion

#region Bad
class_name badClassName
class_name BAD_CLASS_NAME
class_name Bad_Class_Name
#endregion
