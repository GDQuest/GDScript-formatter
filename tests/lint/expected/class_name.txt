tests/lint/input/class_name.gd:6:class-name:error: Class name 'badClassName' should be in PascalCase format
tests/lint/input/class_name.gd:7:class-name:error: Class name 'BAD_CLASS_NAME' should be in PascalCase format
tests/lint/input/class_name.gd:8:class-name:error: Class name 'Bad_Class_Name' should be in PascalCase format